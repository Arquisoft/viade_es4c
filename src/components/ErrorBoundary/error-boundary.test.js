import React from 'react';
import { shallow } from 'enzyme';
import ErrorBoudary from "./error-boundary.component";

describe('ErrorBoundary', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ErrorBoudary debug />);
  
    expect(component).toMatchSnapshot();
  });
});
