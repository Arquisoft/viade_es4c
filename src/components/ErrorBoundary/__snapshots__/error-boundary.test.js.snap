// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should render correctly in "debug" mode 1`] = `""`;
